src/services/invokeService.ts

```diff
@@ -0,0 +1,86 @@
+import { BaseService } from "./baseService"
+import Serverless from "serverless";
+import config from "../config";
+import axios from "axios";
+import { FunctionAppService } from "./functionAppService";
+import { FunctionAppResource } from "../armTemplates/resources/functionApp";
+import { ServerlessAzureConfig } from "../models/serverless";
+
+export class InvokeService extends BaseService {
+  public serverless: Serverless;
+  public options: Serverless.Options;
+  public constructor(serverless: Serverless, options: Serverless.Options) {
+    super(serverless, options);
+    this.serverless = serverless;
+    this.options = options;
+  }
+
+  /**
+   * Invoke an Azure Function
+   * @param functionName Name of function to invoke
+   * @param data Data to use as body or query params
+   * @param method GET or POST
+   */
+  public async invoke(functionName: string, data: any, method: string){
```

Comment:
Consider reordering your params to method, functionName, then data.  Data seems like it should also be optional since not all calls will require data.